### ISSUE TITLE
Tooltips unreadable when using certain themes

### ISSUE THREAD

**Issue by michaelvickersuk** [steelbrain/linter-ui-default#609]
Following updating to version 2.3.1 I've observed the tooltip's text color doesn't have sufficient contrast with the background when using certain themes, resulting in an "empty" looking tooltip.

Screenshot below shows Atom 1.53.0 on Ubuntu 18.04.5 LTS, using Atom Dark UI and syntax themes.

![image](https://user-images.githubusercontent.com/13500452/103095627-2f706380-45f9-11eb-8d55-0f8bb9be44a1.png)

Other theme combinations such as One Light UI and One Dark Syntax also display a similar behaviour.

![image](https://user-images.githubusercontent.com/13500452/103095825-e836a280-45f9-11eb-8db3-13585ffcae41.png)

No errors are shown in the dev tools console when hovering over an error to trigger the tooltip.

Adding the following to my `styles.less` corrects the issue when using the Atom Dark UI and syntax themes.

```css
#linter-tooltip .linter-message.error {
  color: #fff !important;
}
```

Maybe related to the changes from #607 :thinking:

**Comment by aminya** [comment]
Thanks for the report. I fixed it in the new version
